ci-operator/config/openshift/prometheus-community-windows_exporter/openshift-prometheus-community-windows_exporter-release-4.14.yaml

```diff
@@ -0,0 +1,42 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-9-golang-1.20-openshift-4.14:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.20-openshift-4.14
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.20-openshift-4.14
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: build
+  steps:
+    test:
+    - as: test
+      commands: |
+        git clone --branch release-4.14 --single-branch https://github.com/openshift/windows-machine-config-operator.git
+        cd windows-machine-config-operator/
+        git submodule update --init
+        cd promu/
+        GOBIN=/go/src/github.com/openshift/prometheus-community-windows_exporter go install .
+        cd /go/src/github.com/openshift/prometheus-community-windows_exporter
+        GOOS=windows ./promu build -v
```

Comment:
I don't understand this, why is the WMCO repo being cloned into here? For the promu submodule?
If promu is needed, only promu should be gotten.

What if an update to promu is needed to build windows-exporter? You are creating a circular dependency here.

Reply:
>If promu is needed, only promu should be gotten.
True, however there is another element in to consideration; how to sync the `promu` and `prometheus-community-windows_exporter` commit hash WMCO is using at a given time. The later, may explain the need to just pull WMCO code for the given branch

Reply:
>What if an update to promu is needed to build windows-exporter?

That will be resolved within the WMCO repository itself through submodule updates. I don't see any risk for circular dependency